src/Control/UserController.php

```diff
@@ -10,26 +34,25 @@ class UserController extends Controller implements PermissionProvider
         'InvitationForm',
         'AcceptForm',
         'expired',
-        'notfound'
+        'notfound',
     );
 
     public function providePermissions()
     {
-         return array(
+        return array(
             'ACCESS_USER_INVITATIONS' => array(
                 'name' => _t('UserController.ACCESS_PERMISSIONS', 'Allow user invitations'),
-                'category' => _t('UserController.CMS_ACCESS_CATEGORY', 'User Invitations')
-            )
+                'category' => _t('UserController.CMS_ACCESS_CATEGORY', 'User Invitations'),
+            ),
         );
     }
 
-
     public function index()
     {
         if (!Member::currentUserID()) {
-            return $this->redirect('/Security/login');
+            return $this->redirect('Security/login');
```

Comment:
Mind doing here too?
```
$security = Injector::inst()->get(Security::class);
        $this->assertContains("<a href=\"{$baseURL}//Security/login?BackURL=/\">", $body);	        $link = $link $security->Link('login');
```

Reply:
Nice work @Makreig. 

I'll accept as-is and do the minor updates myself.

Thanks.